tests/warp/warp_test.go

```diff
@@ -169,16 +226,21 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		defer sub.Unsubscribe()
 
+		toKey, err := crypto.GenerateKey()
+		gomega.Expect(err).Should(gomega.BeNil())
+		sendToAddress = crypto.PubkeyToAddress(toKey.PublicKey)
+
```

Comment:
We never actually use `toKey` because we only need an address value to include in the contract arguments here and not the key itself.

Could we use a hardcoded address here instead ie. `common.HexToAddress("095e7baea6a6c7c4c2dfeb977efac326af552d87")` so that we don't need this extra code

Reply:
sure!

Reply:
I just left it as a ginkgo test variable